tests/x11/user_defined_snapshot.pm

```diff
@@ -55,6 +55,8 @@ sub run {
     power_action('reboot', keepconsole => 1, textmode => 1);
 
     $self->handle_uefi_boot_disk_workaround() if get_var('MACHINE') =~ qr'aarch64';
+    # post_fail_hook from opensusebasetest triggers if shutdown takes too long time
+    $self->{in_wait_boot} = 1;
```

Comment:
I guess, the `$self->{in_wait_boot} = 1;` line should be before `power_action('reboot'...)`, as assert_shutdown function is located inside of the power_action.

If it will fail on assert_shutdown, then code will never reach the `$self->{in_wait_boot} = 1;` line and `in_wait_boot` in post_fail_hook it will equal to '0'.